modules/ROOT/pages/design-helpful-features-text-editor.adoc

```diff
@@ -82,3 +90,84 @@ types:
           application/json:
             type: person  <3>
 ----
+
+== Find Locations Where a Definition is Referenced
+
+You can navigate to locations where a given definition is referenced by right-clicking on the definition and selecting *Go to References*.
+
+For example, the type `person` is declared on line 5 at the first callout in the following example API specification. That type is subsequently used on line 14 at the second callout, and on line 22 at the third callout. If you right-click `person` on line 5 and select *Go to References*, a list of the `person` locations (callouts two and three) appears. Double clicking on a listed location will navigate you to the relevant line.
```

Comment:
Consider breaking after the second sentence, inserting the syntax (will the audience recognize these as "callouts"?), and then following the syntax with the third and fourth sentences:

"For example,...at the third callout:"
<syntax>
"If you right-click...the relevant line."

Reply:
will navigate you > navigates you

Reply:
Fixed